.jules/runs/surveyor_workspace/decision.md

```
@@ -0,0 +1,18 @@
+# Surveyor Workspace Decision
+
+## Option A
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)